ehsm_kms_service/router.js

```diff
@@ -197,6 +199,37 @@ const router = async (p) => {
       break
     case key_management_apis.DisableKey:
       disableKey(appid, payload, res, DB)
+    case remote_attestation_apis.GenerateQuote:
+      try {
+        const { challenge } = payload
+        if (challenge) {
+          napi_res = napi_result(action, res, [challenge])
+          napi_res && res.send(napi_res)

```

Comment:
@syan10   Is it required to sign the result for GenerateQuote?
Otherwise, the challenge seems useless.

Reply:
OK.  It's not required.  ignore it. Thanks.